labelCloud/model/point_cloud.py

```diff
@@ -59,12 +58,11 @@ def __init__(
         self.path = path
         self.points = points
         self.colors = colors if type(colors) == np.ndarray and len(colors) > 0 else None
-        self.label_definition = label_definition
 
         self.labels = self.label_color_map = None
         if self.SEGMENTATION:
             self.labels = segmentation_labels
-            self.label_color_map = get_distinct_colors(len(label_definition))
+            self.label_color_map = LabelConfig().get_color_map()
```

Comment:
You expect colors with 3 values (RGB). Currently LabelConfig also supports alpha = RGBA. We should decide if that makes sense.

Can also just drop that support.

Reply:
I know changed to RGB only. I think for class colors alpha does not makes sense.